Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt

```diff
@@ -172,12 +187,43 @@ class PersonDetailsFragmentTest : BaseUITest() {
     private fun boosterCertificateData(isCwa: Boolean = false): LiveData<List<CertificateItem>> = MutableLiveData(
         mutableListOf<CertificateItem>().apply {
             val vaccinationCertificate1 = mockVaccinationCertificate(number = 3, final = false, booster = true)
+
             val personCertificates = PersonCertificates(
                 listOf(vaccinationCertificate1),
                 isCwaUser = isCwa
             )
 
+            val stringRes = R.string.vaccination_card_booster_eligible
+
+            val ruleDescriptionDE = mockk<DccValidationRule.Description> {
+                Locale.GERMAN.also {
+                    every { description } returns stringForLocale(it, stringRes)
+                    every { languageCode } returns it.language
+                }
+            }
+
+            val ruleDescriptionEN = mockk<DccValidationRule.Description> {
+                Locale.ENGLISH.also {
+                    every { description } returns stringForLocale(it, stringRes)
+                    every { languageCode } returns it.language
+                }
+            }
+
             add(PersonDetailsQrCard.Item(vaccinationCertificate1, false) {})
+
+            add(
+                VaccinationInfoCard.Item(
+                    vaccinationStatus = VaccinatedPerson.Status.BOOSTER_ELIGIBLE,
+                    daysUntilImmunity = null,
+                    boosterRule = mockk<DccValidationRule>().apply {
+                        every { identifier } returns "AH-1994"
```

Comment:
should we a proper rule id such as `BNR-DE-3009` ? instead of Axel's initials and birthdate 😄

Reply:
:-D
sure